backend/local/backend.go

```diff
@@ -20,8 +19,8 @@ import (
 )
 
 const (
-	DefaultEnvDir          = "terraform.tfstate.d"
-	DefaultEnvFile         = "environment"
+	DefaultWorkspaceDir    = "terraform.tfstate.d"
+	DefaultWorkspaceFile   = "environment"
```

Comment:
I was nervous about touching this right now because it would probably require some careful touches to make sure we don't inadvertently revert people back to `default` when they upgrade. I agree that we'd ideally change it, but my threshold for this initial change was either "easy and low-risk" or "in the UI", and this didn't pass either of those.

Of course, if you disagree with that as the goal then we can talk about that. :grinning: